samples/start-app/php-symfony/tests/UseCase/AppActivationCallbackTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace App\Tests\UseCase;
+
+use App\Entity\Exception\AuthorizationCodeException;
+use App\Entity\Exception\InvalidStateException;
+use App\Entity\Exception\SessionInformationException;
+use App\Entity\Token;
+use App\Repository\TokenRepository;
+use App\Tests\MockApiTrait;
+use App\Tests\Mocks\Oauth2Mock;
+use App\UseCase\AppActivationCallback;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class AppActivationCallbackTest extends TestCase
+{
+
+    use MockApiTrait;
+
+    private AppActivationCallback $appActivationCallback;
+    private MockObject $tokenRepository;
+
+    protected function setUp(): void
+    {
+        $client = new Client(['handler' => $this->mockApi()]);
+        $this->tokenRepository = $this->createMock(TokenRepository::class);
+        $this->appActivationCallback = new AppActivationCallback(
+            $client,
+            $this->tokenRepository,
+            'oauth_client_id',
+            'oauth_client_secret'
+        );
+    }
+
+    /**
+     * @return array
+     */
+    public function getDataStateException() : array
+    {
+        return [
+            array(
```

Comment:
why dont use `[]` instead of `array`

Reply:
It seemed easier to read like this. But changed it anyway.